tasks/ca/statcan/data.py

```diff
@@ -53,6 +53,22 @@ def download(self):
 
 
 class SplitAndTransposeData(BaseParams, Task):
+    INDIAN_RESERVES_SUFFIX = '-DQ'
+    DIVISION_COLUMN_NAMES = {
```

Comment:
* But `StatCanParser` know what to do with columns (for example what columns use, or find or...) maybe this means to move that part too
* Use of a tuple or an object or a merged data structure is to improve readability because two different dictionaries that are related is better to have them merged.
* Regexp would be great to deal with future problems like this. My suggestion is the use regexp in general for this kind of scenarios instead of literal substring so in the future, we are able to execute the regexp which from my point of view is more flexible. I have really bad experiences using substrings (if the Canada census adds a space between the `t` and the parentheses we are in trouble again).
* I'm talking about the line 283-286, extract a method to check for that condition to improve readability